dask/array/tests/test_fft.py

```diff
@@ -18,6 +18,20 @@ def key(k):
     return sorted(a.dask, key=key) == sorted(b.dask, key=key)
 
 
+@pytest.fixture(
+    params=[
+        "fft",
+        "ifft",
+        "rfft",
+        "irfft",
+        "hfft",
+        "ihfft",
+    ]
+)
+def funcname(request):
+    return request.param
```

Comment:
Oh, I'm still in favor of fixtures.  I would just use parametrize.  I don't really care though, both are good.

```python
@pytest.parametrize('funcname', function_names)
def test_foo(funcname)
    ...
```

Totally happy with both though.  I would leave things as is.  I'm just pontificating.